docs/src/prototypes/IconViewer/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+import { Provider, Grid, Divider, Header, Icon } from '@stardust-ui/react'
+import { processedIconsTheme } from 'src/themes/teams/iconTest'
```

Comment:
would suggest to rename `iconTest` to something that would be more expressive, like `withProcessedIcons` (and it would work nice, as path for this file will provide all the necessary context: `src/themes/teams/withProcessedIcons`). As a next step, we might consider to use default export in `withProcessedIcons.ts` to export corresponding theme, so that final solution would be:

```js
// in withProcessedIcons.ts
const theme = { ... }
export default theme

.....

// here, in this file
import themeWithProcessedIcons from 'src/themes/teams/withProcessedIcons'
```